libs/platform/src/lib/components/panel/panel.component.ts

```diff
@@ -0,0 +1,151 @@
+import {
+    Component,
+    Input,
+    ChangeDetectorRef,
+    Output,
+    EventEmitter,
+    ContentChild,
+    ChangeDetectionStrategy,
+    ViewChild,
+    OnInit,
+    OnChanges,
+    SimpleChanges
+} from '@angular/core';
+
+import { PanelTitleDirective } from '@fundamental-ngx/core';
+
+import { BaseComponent } from '../base';
+import { ContentDensity } from '../form/form-control';
+
+import { PanelConfig } from './panel.config';
+import { PanelActionsComponent } from './panel-actions/panel-actions.component';
+import { PanelContentComponent } from './panel-content/panel-content.component';
+
+/** Panel change event instance */
+export class PanelExpandChangeEvent {
+    constructor(public source: PanelComponent, public payload: boolean) {}
+}
+
+/**
+ * Fundamental Panel component
+ *
+ * ```html
+ * <fdp-panel title="Panel Header">
+ *     <fdp-panel-content>
+ *        Panel Content
+ *     </fdp-panel-content>
+ * </fdp-panel>
+ * ```
+ *
+ * */
+@Component({
+    selector: 'fdp-panel',
+    templateUrl: './panel.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PanelComponent extends BaseComponent implements OnInit, OnChanges {
+    /**
+     * sets Panel title.
+     */
+    @Input()
+    title: string;
+
+    /**
+     * Whether the Panel Content is expanded
+     */
+    @Input()
+    expanded = true;
+
+    /**
+     * Whether the Panel is expandable
+     */
+    @Input()
+    expandable = true;
+
```

Comment:
We are still adding types even if a value is assigned. At least in Core :)

Reply:
Recently new `no-inferrable-types` ts-lint rule has been added ([PR](https://github.com/SAP/fundamental-ngx/pull/2933))
From that moment we do not need to write types in such cases like this one, more over it will warning us that is redundant.